tests/check_vcs_permalinks_test.py

```diff
@@ -33,3 +33,43 @@ def test_failing(tmpdir, capsys):
             'Non-permanent github link detected.\n'
             'On any page on github press [y] to load a permalink.\n'
         )
+
+
+def test_passing_with_github_enterprise(tmpdir):
+    f = tmpdir.join('f.txt')
+    f.write_binary(
+        # permalinks are ok
+        b'https://example.com/asottile/test/blob/649e6/foo%20bar#L1\n'
+        # links to files but not line numbers are ok
+        b'https://example.com/asottile/test/blob/master/foo%20bar\n'
+        # regression test for overly-greedy regex
+        b'https://example.com/ yes / no ? /blob/master/foo#L1\n',
+    )
```

Comment:
you can just use a simpler test, no need to re-test everything again

Reply:
Since the logic does not change even if the user specifies additional domains to check, I added a case that fails. Are there any more test cases to add?